tests/test_external/test_sdk/test_bell_corrections.py

```diff
@@ -0,0 +1,170 @@
+import numpy as np
+import pytest
+
+from netqasm.lang.instr.core import LoadInstruction, SetInstruction
+from netqasm.lang.operand import Immediate
+from netqasm.lang.subroutine import Subroutine
+from netqasm.logging.glob import get_netqasm_logger
+from netqasm.qlink_compat import BellState
+from netqasm.runtime.application import default_app_instance
+from netqasm.runtime.settings import Simulator, get_simulator
+from netqasm.sdk import EPRSocket
+from netqasm.sdk.external import NetQASMConnection, simulate_application
+
+logger = get_netqasm_logger()
+
+
+def simulate_different_bell_state(
+    subrt: Subroutine, bell_state: BellState
+) -> Subroutine:
+    # Change "Load bell state index from array" instruction into a simple "set"
+    orig_instr: LoadInstruction = subrt.instructions[27]  # type: ignore
```

Comment:
I guess the index 27 (?) should change when the instruction generator is changed. Is this the best way to find the instruction to change?

Reply:
Good point. I changed it so that it finds the second 'load' instruction rather than using a hardcoded instruction index.